### DISCUSSION TITLE
Month string placement in date range literals

### DISCUSSION THREAD

**Discussion by asieraduriz** [js-fns/date-fns#2825] [General]
For the english version of:
`December 22-28`
Wrong russian:
`ноября 22-28`
Right russian:
`22-28 ноября`

Is there something in the API that gets two `Date` objects and might return the placement of the month in the appropriate place?

**Comment by fturmel** [comment]
There are no range formatting functions in `date-fns` as far as I know.

You can try using `Intl` for this, it didn't take me long to get the exact localized output you're looking for.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/formatRange

```js
function formatRange(d1, d2, locale) {
  return new Intl.DateTimeFormat(locale, {
    month: "long",
    day: "numeric"
  }).formatRange(d1, d2);
}

const start = new Date(2021, 11, 22);
const end = new Date(2021, 11, 28);

console.log(formatRange(start, end, "en")); // December 22 – 28
console.log(formatRange(start, end, "ru")); // 22–28 декабря
````
